devolutions-gateway/src/api/network_scan/mod.rs

```diff
@@ -0,0 +1,12 @@
+use crate::DgwState;
+use axum::Router;
+
+pub mod ipconfig;
+pub mod scan;
+
+pub fn make_router<S>(state: DgwState) -> Router<S> {
+    Router::new()
+        .route("/", axum::routing::get(scan::handler))
+        .route("/ipconfig", axum::routing::get(ipconfig::handler))
+        .with_state(state)
```

Comment:
suggestion: If we’re going to have nested routes like this, I would suggest something like this:

- `/jet/net/scan`: Run the network scanning (WebSocket)
- `/jet/net/config`: Fetch the network interfaces configuration (plain HTTP call)

Reply:
See https://github.com/Devolutions/devolutions-gateway/pull/709

Reply:
updated